lsp/io.ml

```diff
@@ -56,7 +77,6 @@ let mut = Mutex.create ()
 
 let send_json fmt obj =
   Mutex.lock mut;
-  if Fleche.Debug.send then Log.log_object "send" obj;
```

Comment:
Is the option unused now?

Reply:
No, but setting `coq-lsp.trace.server` to `verbose` supersedes this.